integration_tests/.scripts/integration_test.sh

```diff
@@ -38,15 +39,29 @@ for db in ${DATABASES[@]}; do
   echo "Snowplow Fractribution integration tests: Execute fractribution models"
 
   eval "dbt run --select snowplow_fractribution --target $db --full-refresh" || exit 1;
```

Comment:
I think the main run of the package needs to have the model variable as well, otherwise it's always going to run with the default model and then the test views will be based on a different one?

Reply:
I changed this - thanks!